iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/PhaseShifterShiftTap.java

```diff
@@ -55,4 +55,23 @@ private void adjustTapPosition(PhaseTapChanger phaseTapChanger) {
         phaseTapChanger.setTapPosition(Math.min(Math.max(phaseTapChanger.getTapPosition() + tapDelta,
                 phaseTapChanger.getLowTapPosition()), phaseTapChanger.getHighTapPosition()));
     }
+
+    @Override
+    public NetworkModificationImpact hasImpactOnNetwork(Network network) {
```

Comment:
Indeed I don't understand why we have this `PhaseShifterShiftTap` as we can use the generic `PhaseTapPositionModification`... this is not the object of the PR, but not really understandable.